src/main/java/school/lemon/changerequest/java/introduction/hw1/Task9.java

```diff
@@ -3,6 +3,21 @@
 public class Task9 {
 
     public static boolean isPowerOfThree(int n) {
-        return false;
+        return verifyIsPowerOfThree(n);
+    }
+
+    private static boolean verifyIsPowerOfThree(int n) {
+        if (n == 0) {
```

Comment:
By the way, you are doing such things via recursion. 
Have you tried to do it via loop? I'd prefer loops, but no need to change.